CONTRIBUTING.md

```diff
@@ -85,13 +85,13 @@ Available types:
 We welcome everyone to review Pull Requests, it is a great way to learn and network.
 
 ##### DOs
-- Be kind and respectful, we use inclusive, gender neutral language (for example they/them instead of guy/man)
+- Be kind and respectful, we use inclusive, gender-neutral language (for example they/them instead of guy/man)
 - Use inline comments to explain your suggestions
 - Use inline suggestions to propose changes
 ##### DON'Ts
 - Don't be rude, disrespectful or aggressive
 - Do not repeat feedback, this creates more noise than value (check the existing conversation), use GitHub reactions if you agree/disagree with a comment
-- Do not blindly approve pull requests to improve your GitHub contributors graph
+- Do not blindly approve pull requests to improve your GitHub contributor's graph

```

Comment:
This was a mistake on my side. Can you please change `contributor's` to `contribution`